mobile/js_files/package.json

```diff
@@ -22,13 +22,12 @@
     "emojilib": "^2.4.0",
     "eth-phishing-detect": "^1.1.13",
     "functional-red-black-tree": "^1.0.1",
-    "hermes-engine": "0.2.1",
```

Comment:
Is hermes now pulled in by `react-native` or what?

Reply:
Yes, they are dependent on each other and if they are not in sync the android build does not work. It was hardcoded here because when react-native 60 was using a version with a bug and it was overridden to fix it. More info there: https://github.com/status-im/status-react/pull/9001

Reply:
Also new version available, and will come with next react-native version https://github.com/facebook/hermes/releases/tag/v0.5.0

Reply:
Sweet, thanks for explaining.